src/test/java/org/opentripplanner/util/time/TimeUtilsTest.java

```diff
@@ -47,6 +47,17 @@ public void timeToStrCompact() {
     assertEquals("10:26:03-1d", TimeUtils.timeToStrCompact(-T13_33_57));
   }
 
+  @Test
+  public void timeToStrCompactNoSeconds() {
+    assertEquals("0:00", TimeUtils.timeToStrCompactNoSeconds(T00_00_01));
+    assertEquals("0:01", TimeUtils.timeToStrCompactNoSeconds(T00_01_00));
+    assertEquals("1:00", TimeUtils.timeToStrCompactNoSeconds(T01_00_00));
+
+    assertEquals("13:33", TimeUtils.timeToStrCompactNoSeconds(T13_33_57));
+    assertEquals("0:58", TimeUtils.timeToStrCompactNoSeconds(T00_58_59));
+    assertEquals("", TimeUtils.timeToStrCompact(NOT_SET, NOT_SET));
```

Comment:
This is testing different method. Should this test belong somewhere else?

Reply:
It was a copy-and-paste mistake. However, it made me realise that there is the option of being called with two parameters which I didn't implement.

So, to sidestep the discussion whether the two arguments version needs to be implemented, I moved the truncation code to inside the domain class.

You could argue that it's the correct place anyway.